nix/nixos/cardano-cluster-service.nix

```diff
@@ -7,7 +7,7 @@ with lib; with builtins;
 let
   cfg = config.services.cardano-cluster;
   ncfg = config.services.cardano-node;
-  node-ids = range 0 (cfg.node-count - 1);
+  node-ids = range 1 (cfg.node-count);
```

Comment:
This change is breaking the topology generator, @karknu -- let's discuss if we really need the numbering change.

Reply:
Removed.